provision/rosa-cross-dc/keycloak-benchmark-crossdc-tests/src/test/java/org/keycloak/benchmark/crossdc/FailoverTest.java

```diff
@@ -31,4 +61,61 @@ public void logoutUserWithFailoverTest() throws IOException, URISyntaxException,
 
         LOAD_BALANCER_KEYCLOAK.refreshToken(REALM_NAME, (String) tokensMap.get("refresh_token"), CLIENTID, CLIENT_SECRET, 400);
     }
+
+    @Test
+    @ActiveActive
+    public void ensureAcceleratorUpdatedOnSplitBrainTest() throws Exception {
```

Comment:
+1 I think that would be a good test to have. Maybe in a follow-up PR though?

I think we would need to add an additional REST resource to the dataset so that we can determine from REST calls made via the Accelerataor DNS which site is actually handling the request. As the Accelerator is not DNS based, we can't query the IP to determine which site(s) are responding to requests.

Reply:
I created a task on our board for this.